modules/effects/src/effect_sources.ts

```diff
@@ -72,11 +72,8 @@ export class EffectSources extends Subject<any> {
 }
 
 function effectsInstance(sourceInstance: any) {
-  if (
-    onIdentifyEffectsKey in sourceInstance &&
-    typeof sourceInstance[onIdentifyEffectsKey] === 'function'
-  ) {
-    return sourceInstance[onIdentifyEffectsKey]();
+  if (isOnIdentifyEffects(sourceInstance)) {
+    return sourceInstance.ngrxOnIdentifyEffects();
```

Comment:
could you please add `declare` to all interfaces (`OnIdentifyEffects`, `OnRunEffects ` and `OnInitEffects `), so they are not property-renamed and we are able to call these methods on the instance.

Reply:
@alex-okrushko I can, and I did 🙂

Reply:
@alex-okrushko out of curiosity, I don't end up of understand why `declare` keyword is needed with interfaces :thinking:

Reply:
Using `declare` keyword is helping TS mark an interface as external. This will help some JS Compilers (for example like [Closure Compiler](https://developers.google.com/closure/compiler)) that do property renaming, **not to rename these fields**. 
e.g. object with prop rename would turn `{ngrxOnIdentifyEffects: () => doStuff()}` to `{bds: () => aef()}` which helps reduce the final bundle size.

Why would we care if these are renamed in this case?
We are [passing function names](https://github.com/ngrx/platform/pull/2320/files#diff-7f170bfaeb28b7144e3545dbe130a918R127) as strings, and string are not renamed, so we would attempt to check/call `obj['ngrxOnIdentifyEffects']` while the prop was renamed to `bds` and so it won't be found, causing a bug (or unexpected behavior).